python/Ganga/GPIDev/Adapters/IGangaFile.py

```diff
@@ -71,6 +111,14 @@ def put(self):
         """
         raise NotImplementedError
 
+    def copyTo(self, targetPath):
+        """
+        Copy a the file to the local storage using the appropriate file-transfer mechanism
+        Args:
+            targetPath (str): Target path where the file is to copied to
+        """
+        raise NotImplementedError
```

Comment:
I'd argue it makes much more sense to implement this particular functionality for every file object it's the basic requirement of using your internal structure place me a file to a given location. The get method is simply an extension of this with helpers for the rest of Ganga built-in.